cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/query_enhancements/simple_dataset_selector.spec.js

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  INDEX_PATTERN_WITH_TIME,
+  INDEX_WITH_TIME_1,
+  INDEX_PATTERN_WITH_NO_TIME,
+  INDEX_WITHOUT_TIME_1,
+  SECONDARY_ENGINE,
+} from '../../../../../utils/constants';
+import {
+  getRandomizedWorkspaceName,
+  getRandomizedDatasourceName,
+  getDefaultQuery,
+  setDatePickerDatesAndSearchIfRelevant,
+} from '../../../../../utils/apps/query_enhancements/shared';
+import { verifyDiscoverPageState } from '../../../../../utils/apps/query_enhancements/saved';
+import {
+  generateSimpleDatasetSelectorTestConfigurations,
+  validateItemsInSimpleDatasetSelectorDropDown,
+} from '../../../../../utils/apps/query_enhancements/simple_dataset_selector';
+
+const workspaceName = getRandomizedWorkspaceName();
+const datasourceName = getRandomizedDatasourceName();
```

Comment:
(no action needed): I created this function, but I regret doing so now after seeing how neo handles things :/ . I don't think we should use it (but don't worry about it, i was planning on doing a PR that removes all usage of it later)